morphic.js

```diff
@@ -8047,22 +8131,45 @@ StringMorph.prototype.deleteSelection = function () {
 };
 
 StringMorph.prototype.selectAll = function () {
+    var cursor;
     if (this.isEditable) {
         this.startMark = 0;
+        cursor = this.root().cursor;
```

Comment:
Whoa! Why do we need to move the cursor here? What if we have several ```StringMorph```s or ```TextMorph```s with different selections... Oh. Maybe we never do that... ?